galaxykubeman/templates/rbac.yaml

```diff
@@ -0,0 +1,30 @@
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: {{ .Release.Namespace }}-rbac
+  namespace: {{ .Release.Namespace }}
+rules:
+  - apiGroups: ["", "extensions", "apps"]
+    resources: ["*"]
+    verbs: ["*"]
+  - apiGroups: ["batch"]
+    resources:
+      - jobs
+      - cronjobs
+    verbs: ["*"]
+
+---
+
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: {{ .Release.Namespace }}-rbac-binding
+  namespace: {{ .Release.Namespace }}
+subjects:
+  - kind: ServiceAccount
+    name: {{ .Values.rbac.ksa }}
```

Comment:
Could we change this to `.rbac.serviceAccount` if there are no objections?

(That's the standard we use on our charts: https://github.com/CloudVE/cloudman-helm/blob/master/cloudman/values.yaml#L1278 and will be the name for all other GalaxyKubeman charts as well)